app/money.cpp

```diff
@@ -3,5 +3,74 @@
 
 namespace vsite::oop::v7
 {
+	double operator ""_EUR(unsigned long long price) {
+		return price / 7.53;
+	}
+
+	double operator ""_USD(unsigned long long price) {
+		return price / 6.98;
```

Comment:
takozvani fiksni tečaj :relaxed: ?

Reply:
A to sam dodala čisto da ima neki dodatni test konverzije, budući da ne znam kako bih napravila da mi od negdje vuče tečaj, ali mogu uvesti varijablu pa da korisnik unosi.